modules/tinymce/src/themes/silver/main/ts/ui/core/complex/BespokeNumberInput.ts

```diff
@@ -132,6 +137,7 @@ const createBespokeNumberInput = (editor: Editor, backstage: UiFactoryBackstage,
           ]),
           AddEventsBehaviour.config('input-update-display-text', [
             AlloyEvents.run<UpdateMenuTextEvent>(updateMenuText, (comp, se) => {
+              oldValue = Optional.some(se.event.text);
```

Comment:
I actually didn't do it because I wasn't sure how to do so, the only way that I found is to use `editor.queryCommandValue('FontSize')`, similar to how it is calculated in [FontSizeBespoke.ts](https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/themes/silver/main/ts/ui/core/complex/FontSizeBespoke.ts#L142), and I thought that it would be incorrect since this is `BespokeNumberInput` and it shouldn't be bound to the use of the `font size` (it could be any other kind of number) but then I noticed that we already use the `Dimension.parse` which bind the entire component to the fact that we are using it for the `font size`, now I see these possible ways to solve it:

1. rename the `BespokeNumberInput` in `BespokeFontSizeInput` and use `editor.queryCommandValue('FontSize')`
2. find a way to also remove the `Dimension.parse` and don't use it nor `editor.queryCommandValue('FontSize')`
3. use `editor.queryCommandValue('FontSize')` and when we will need to use this component for a different kind of number, refactor it

Reply:
ok, in the end the split the `FontSize` functionality from `BespokeNumberInput`, I move the parse logic from in `FontSizeBespoke.ts`.

So now `validation` and `defaultValue` is demanded to `FontSizeBespoke.ts` and `BespokeNumberInput` require only a `getNewValue` function